WebDevStudios/nextjs-wordpress-starter
# Feature/bulk dependabot 2021 06 07

Closes #499
Closes #500
Closes #501
Closes #502
Closes #503

### Description

Bump the following Node dependencies:

```bash
 @arkweid/lefthook            ^0.7.5  →    ^0.7.6     
 @storybook/addon-a11y        ^6.3.0-alpha.36  →  ^6.3.0-beta.15     
 @storybook/addon-essentials  ^6.3.0-alpha.36  →  ^6.3.0-beta.15     
 @storybook/addon-links       ^6.3.0-alpha.36  →  ^6.3.0-beta.15     
 @storybook/builder-webpack5  ^6.3.0-alpha.36  →  ^6.3.0-beta.15     
 @storybook/manager-webpack5  ^6.3.0-alpha.36  →  ^6.3.0-beta.15     
 @storybook/react             ^6.3.0-alpha.36  →  ^6.3.0-beta.15     
 @testing-library/jest-dom    ^5.12.0  →   ^5.13.0     
 chromatic                     ^5.8.3  →    ^5.9.0     
 eslint                       ^7.27.0  →   ^7.28.0     
 eslint-plugin-jsdoc          ^35.1.2  →   ^35.1.3     
 formik                        ^2.2.8  →    ^2.2.9     
 jest                         ^27.0.3  →   ^27.0.4     
 next-sitemap                 ^1.6.99  →  ^1.6.105     
 prettier                      ^2.3.0  →    ^2.3.1     
 tailwindcss                   ^2.1.2  →    ^2.1.4     
 next-auth                    ^3.24.0  →   ^3.25.0   
```

### Screenshot

Frontend:

![screenshot](https://dl.dropbox.com/s/skx5t5zap3mq9en/Screen%20Shot%202021-06-07%20at%2009.33.48.png?dl=0)

Storybook:

![screenshot](https://dl.dropbox.com/s/u63h5ks9drf9n2m/Screen%20Shot%202021-06-07%20at%2009.36.19.png?dl=0)

### Verification

How will a stakeholder test this?

1. `gh pr checkout 504`
2. `npm ci`
3. `npm run build && npm run start`
4. `npm run storybook`
5. Verify the frontend loads
6. Verify Storybook loads
